src/reportengine/environment.py

```diff
@@ -43,6 +44,11 @@ def __init__(self, *, output=None, formats=('pdf',),
         self.loglevel = loglevel
         self.extra_args = kwargs
         self.config_yml = config_yml
+        if folder_prefix and config_yml:
```

Comment:
Out of interest when will config_yml be false? When using the API?

Reply:
Yes, for example.